articles/azure-monitor/containers/container-insights-agent-config.md

```diff
@@ -34,6 +34,7 @@ The following table describes the settings you can configure to control data col
 | `[log_collection_settings.env_var] enabled =` | Boolean | True or false | This setting controls environment variable collection<br> across all pods/nodes in the cluster<br> and defaults to `enabled = true` when not specified<br> in the ConfigMap.<br> If collection of environment variables is globally enabled, you can disable it for a specific container<br> by setting the environment variable<br> `AZMON_COLLECT_ENV` to `False` either with a Dockerfile setting or in the [configuration file for the Pod](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/) under the `env:` section.<br> If collection of environment variables is globally disabled, you can't enable collection for a specific container. The only override that can be applied at the container level is to disable collection when it's already enabled globally. |
 | `[log_collection_settings.enrich_container_logs] enabled =` | Boolean | True or false | This setting controls container log enrichment to populate the `Name` and `Image` property values<br> for every log record written to the **ContainerLog** table for all container logs in the cluster.<br> It defaults to `enabled = false` when not specified in the ConfigMap. |
 | `[log_collection_settings.collect_all_kube_events] enabled =` | Boolean | True or false | This setting allows the collection of Kube events of all types.<br> By default, the Kube events with type **Normal** aren't collected. When this setting is set to `true`, the **Normal** events are no longer filtered, and all events are collected.<br> It defaults to `enabled = false` when not specified in the ConfigMap. |
+| `[log_collection_settings.enable_multiline_logs] enabled =` | String | True or False | This setting controls whether multiline container logs are enabled. They are disabled by default. [Click here](https://learn.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-logging-v2?tabs=configure-portal#multi-line-logging-in-container-insights-preview) to learn more about Multi-line logging. |
```

Comment:
```suggestion
| `[log_collection_settings.enable_multiline_logs] enabled =` | String | True or False | This setting controls whether multiline container logs are enabled. They are disabled by default. See [Multi-line logging in Container Insights (preview)](/azure/azure-monitor/containers/container-insights-logging-v2?tabs=configure-portal#multi-line-logging-in-container-insights-preview) to learn more about Multi-line logging. |
```

Reply:
@ibrar2411 should we remove (Preview)?

Reply:
> 

@pfrcks , we have removed the (Preview) .
@Court72 , updated the PR with your suggestions